src/test/scala/firrtlTests/ParserSpec.scala

```diff
@@ -21,7 +21,7 @@ class ParserSpec extends FirrtlFlatSpec {
                      "readwriter" -> "c"
                     )
     def fieldsToSeq(m: Map[String, String]): Seq[String] =
-      m map { case (k,v) => s"      ${k} => ${v}" } toSeq
```

Comment:
no need to change here.

Reply:
I needed to change the `} toSeq` to `}.toSeq` because `2.13` is stricter about postfix notation.
I then also changed the `map` to use the now recommended `.`.